src/components/references/ReferenceLine.tsx

```diff
@@ -0,0 +1,40 @@
+/*!
+ * © 2023 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react'
+import styled from 'styled-components'
+
+import { BibliographyItemAttrs, metadata } from '../../lib/references'
+
+export const Metadata = styled.div`
+  color: ${(props) => props.theme.colors.text.secondary};
+  flex: 1;
+  font-weight: ${(props) => props.theme.font.weight.light};
+  margin-top: ${(props) => props.theme.grid.unit}px;
+`
+
+export const MetadataContainer = styled.div`
+  flex: 1;
+`
+
+export const ReferenceLine: React.FC<{
+  item: BibliographyItemAttrs
+}> = ({ item }) => (
+  <MetadataContainer>
+    <div>{item.title || 'Untitled'}</div>
```

Comment:
this will be like this for some reference with marks:
![Screenshot from 2024-04-16 23-08-01](https://github.com/Atypon-OpenSource/manuscripts-body-editor/assets/45534101/392c84b6-14da-4704-af68-eeb3aaecb653)

is that ok? it was wrapped before with `<Title value={item.title || 'Untitled'} title={item.title} />` to render marks.

and the Title is a component in `manuscripts-title-editor`

Reply:
Yes, the title editor was removed from here a while back. I'm just copying the component from style-guide.